graph/global-defaults.yaml

```diff
@@ -0,0 +1,39 @@
+# Convenience definitions, use as eg. $commit
+commit: "{{.Run.Commit}}"
+repository: "{{.Run.Repository}}"
+branch: "{{.Run.Branch}}"
+gitTag: "{{.Run.GitTag}}"
+triggeredBy: "{{.Run.TriggeredBy}}"
+registry: "{{.Run.Registry}}"
+registryName: "{{.Run.RegistryName}}"
+date: "{{.Run.Date}}"
+sharedVolume: "{{.Run.SharedVolume}}"
+os: "{{.Run.OS}}"
+architecture: "{{.Run.Architecture}}"
+id: "{{.Run.ID}}"
+osArchitecture: ""
```

Comment:
Please reference to [https://github.com/Azure/acr-builder/blob/master/docs/templates.md#run-variables](https://github.com/Azure/acr-builder/blob/master/docs/templates.md#run-variables)

Reply:
Ill match the names put forth there